include/ofi.h

```diff
@@ -83,6 +83,8 @@ extern "C" {
 
 #define ofi_div_ceil(a, b) ((a + b - 1) / b)
 
+#define OFI_MAGIC_64 (0xabcdef0123456789)
```

Comment:
Let's make the magic more random and ensure that it won't point to a valid address